content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,130 +1,60 @@
-```
-
-> See also `dvc.lock`.
-
-## Switching versions
-
-After iterating on this process and producing several versions, you can combine
-`git checkout` and `dvc checkout` to perform full or partial
-<abbr>workspace</abbr> restorations.
-
-![](/img/versioning.png) _Code and data checkout_
-
-> Note that `dvc install` enables auto-checkouts of data after `git checkout`.
-
-A full checkout brings the whole <abbr>project</abbr> back to a previous version
-— code, dataset and model files all match each other:
-
-```dvc
-$ git checkout v1.0
-$ dvc checkout
-M       images
-M       model.pkl
-```
-
-However, we can checkout certain parts only, for example if we want to keep the
-latest source code and model versions, but rewind to the previous version of the
-dataset:
-
-```dvc
-$ git checkout v1.0 images.dvc
-$ dvc checkout images.dvc
-M       images
-```
-
-DVC [optimizes](/doc/user-guide/large-dataset-optimization) this operation by
-avoiding copying files each time, so checking out data is quick even if you are
-versioning large data files.
+# Versioning Data and Models
+
+Data science teams today face data management questions around versioning
+datasets, data artifacts, and machine learning models. How do we keep track of
+changes in data, code, and ML models together? What's the best way to organize
+and store multiple versions of data and model files? How can data lifecycles be
+defined and enforced?
+
+![](/img/data-ver-complex.png) _Exponential complexity of data science projects_
+
+Data Version Control (DVC) can address these issues by helping you organize your
+data and models effectively and capture their versions in
+[Git commits](<(https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository)>),
+while storing the actual data contents outside ot Git. This way there's a single
+project history for all of its assets and artifacts, and at the same time the
+data stays with you.
+
+![](/img/project-versions.png) _DVC matches the right versions of code and
+data._
+
+DVC enables data _versioning through codification_. This means describing which
+data, ML artifacts, etc. should be in the environment at any given time. This is
+achieved with special [metafiles](/doc/user-guide/dvc-files-and-directories)
+that can be put in Git — but we won't much deeper into how it works here, since
+we have many [guides](/doc/user-guide) and [references](/doc/command-reference)
+for that. To learn how it looks and feels, try our
+[versioning tutorial](/doc/use-cases/versioning-data-and-model-files/tutorial)
+👩‍💻.
+
+DVC's approach brings the following benefits:
+
+- **Tidy project**: Work with a natural file structure. No need for ad hoc
+  naming conventions like `data/20190922/labels_v7_final`.
+- **Reproducibility**: Restore any project version and find the corresponding
+  data instantly. Identify past research inputs to understand its results, or
+  for debugging.
+- **Easy storage**: All versions of data are
```

Comment:
@shcheklein here's the latest list of benefits with the advanced scenarios included. Known problems:
- It's too long again.
- Not sure about "Integration" but can't think of a better term rn.

PTAL.